modules/gocd/conf/cruise-config.xml

```diff
@@ -141,6 +141,45 @@
           </job>
         </jobs>
       </stage>
+      <stage name="Deploy-to-Dev" fetchMaterials="false">
+        <jobs>
+          <job name="Run">
+            <tasks>
+              <exec command="/bin/bash" workingdir="#{SOURCE_DIR}">
+                <arg>-c</arg>
+                <arg>git config --global user.email "you@example.com"
+                     git config --global user.name "Your Name"</arg>
+                <runif status="passed" />
+              </exec>
+              <exec command="/bin/bash" workingdir="#{SOURCE_DIR}">
+                <arg>-c</arg>
+                <arg>if [ ! -d "/app/app-dev-unit" ]; then sudo mkdir -p /app/app-dev-unit; sudo chown go:go /app/app-dev-unit;fi; \
```

Comment:
@hazim1093 i think we shld rename this repo "app-dev-unit"; why have the word "dev"... this is generic